css/media.css

```diff
@@ -0,0 +1,321 @@
+@media(min-width: 320px) {
+    body {
+        box-sizing: border-box;
+        font-family: Montserrat, sans-serif;
+        background-color: #fff;
+    }
+    .header {
+        display: grid;
+        grid-template-columns: 4fr 4fr 4fr;
+        align-items: center;
+        justify-content: center;
+        margin-bottom: 30px;
+    }
+    .header-nav {
+        visibility: hidden;
+    }
+    .header-menu__link {
+        visibility: hidden;
+    }
+    .header-menu {
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+    }
+    .header-menu__mobule {
+        position: relative;
+        width: 24px;
+        height: 2px;
+        background-color: #E51937;
+    }
+    .header-menu__mobule::before {
+        content: ' ';
+        position: absolute;
+        width: 24px;
+        height: 2px;
+        background-color: #E51937;
+        top: -6px;
+    }
+    .header-menu__mobule::after {
+        content: ' ';
+        position: absolute;
+        width: 24px;
+        height: 2px;
+        background-color: #E51937;
+        bottom: -6px;
+    }
+    .main-content {
+        background: url('../main-content/Tesla-Wordmark-Red.png');
+        max-height:  467px;
+        background-size: 100%;
+        background-position: center;
+        display: grid;
+        grid-template-rows: repeat(3, 1fr);
+        margin-bottom: 30%;
+    }
+    .main-content__logo {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla_car/pullRequest?prid=5929182)